wp-content/themes/delta-child/functions.php

```diff
@@ -102,12 +102,13 @@ function ajax_projects_pagination()
 add_action('wp_ajax_ajax_projects_pagination', 'ajax_projects_pagination');
 // end
 
-function getFirstParagraph($text) {
+function getFirstParagraph($text)
+{
     // Use a regular expression to split the text into paragraphs
     $paragraphs = preg_split('/\r\n|\r|\n/', $text);
 
     // Remove empty paragraphs that may have been caused by extra line breaks
-    $paragraphs = array_filter($paragraphs, function($para) {
+    $paragraphs = array_filter($paragraphs, function ($para) {
```

Comment:
why is there a space betweeen function and  ($para)

Reply:
For readability, which is pretty normal in this case